hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -679,32 +679,60 @@ public AuthType getAuthType() {
 
   @VisibleForTesting
   String initializeUserAgent(final AbfsConfiguration abfsConfiguration,
-                             final String sslProviderName) {
+      final String sslProviderName) {
+
     StringBuilder sb = new StringBuilder();
-    sb.append("(JavaJRE ");
+
```

Comment:
can we have org.apache.hadoop.util.VersionInfo,getVersion() in there too? Lets people pin the blame on the logs

Reply:
CLIENT_VERSION includes org.apache.hadoop.util.VersionInfo,getVersion()